src/System.Windows.Forms/src/System/Windows/Forms/CreateParams.cs

```diff
@@ -72,26 +70,7 @@ public class CreateParams
 
         public override string ToString()
         {
-            var builder = new StringBuilder(64);
-            builder.Append("CreateParams {'");
-            builder.Append(ClassName);
-            builder.Append("', '");
-            builder.Append(Caption);
-            builder.Append("', 0x");
-            builder.Append(Convert.ToString(Style, 16));
-            builder.Append(", 0x");
-            builder.Append(Convert.ToString(ExStyle, 16));
-            builder.Append(", {");
-            builder.Append(X);
-            builder.Append(", ");
-            builder.Append(Y);
-            builder.Append(", ");
-            builder.Append(Width);
-            builder.Append(", ");
-            builder.Append(Height);
-            builder.Append('}');
-            builder.Append('}');
-            return builder.ToString();
+            return $"CreateParams {{'{ClassName}', '{Caption}', 0x{Style:x}, 0x{ExStyle:x}, {{{X}, {Y}, {Width}, {Height}}}}}";
```

Comment:
If I am not mistaken I think there are some extra curly braces here
```suggestion
            return $"CreateParams {'{ClassName}', '{Caption}', 0x{Style:x}, 0x{ExStyle:x}, {{X}, {Y}, {Width}, {Height}}}";
```

Reply:
In the original string there are 2 curly braces. Here it's interpolated so I have to double them to escape them, right? So 4 curly braces, plus the one that wraps the variable. Here is a small [repro](https://sharplab.io/#v2:C4LgTgrgdgNAJiA1AHwAICYAMBYAUBgRjzwBEB7CAIwBsBTAAgAlaBLAcwAth6BeeggHQBWOAG48AZWBgWUNvQDC1AIYBnVQDllAWwZ8ARADEyZfePwEAnAAoAJPoVhay4LQAKysDtX0A3r4ByXyU1TR1aAF8AmD9mdi4IxMT9AEpRIA)

I agree it doesn't read well. I considered changing it to a raw string, but I liked that it was a one liner.

Reply:
Ah yes it was my mistake 😄 , apologies